app/assets/stylesheets/marking_schemes.scss

```diff
@@ -0,0 +1,17 @@
+@import 'common/core';
+
+table {
```

Comment:
@david-yz-liu There is, but it requires the `<table>` element to be wrapped in `<div class='table'></div>` to allow for horizontal scrolling on smaller screens.